numpy/core/numeric.pyi

```diff
@@ -378,18 +409,24 @@ def outer(
     out: None = ...,
 ) -> NDArray[timedelta64]: ...
 @overload
-def outer(
-    a: _ArrayLikeObject_co,
-    b: _ArrayLikeObject_co,
-    out: None = ...,
-) -> NDArray[object_]: ...
-@overload
 def outer(
     a: _ArrayLikeComplex_co | _ArrayLikeTD64_co | _ArrayLikeObject_co,
     b: _ArrayLikeComplex_co | _ArrayLikeTD64_co | _ArrayLikeObject_co,
     out: _ArrayType,
 ) -> _ArrayType: ...
+@overload
+def outer(
+    a: _ArrayLikeObject_co,
+    b: _ArrayLikeObject_co,
+    out: None = ...,
+) -> NDArray[object_]: ...
```

Comment:
This overload should return to its previous position; I assume this is a leftover edit from something?